pylops/signalprocessing/dct.py

```diff
@@ -0,0 +1,137 @@
+__all__ = ["DCT"]
+
+from typing import List, Optional, Union
+
+import numpy as np
+from scipy import fft
+
+from pylops import LinearOperator
+from pylops.utils._internal import _value_or_sized_to_tuple
+from pylops.utils.decorators import reshaped
+from pylops.utils.typing import DTypeLike, InputDimsLike, NDArray
+
+
+class DCT(LinearOperator):
+    r"""Discreet Cosine Transform
+
+    Performs discreet cosine transform on the given multi-dimensional
```

Comment:
This is a bit too short, see https://pylops.readthedocs.io/en/stable/api/generated/pylops.signalprocessing.FFT.html#pylops.signalprocessing.FFT for an example.. we should at least mention we use scipy dct/idct functions

Reply:
fixed in [289d51a](https://github.com/PyLops/pylops/pull/480/commits/289d51ad61c7294451d777af6367ea282ad1e78e)